### DISCUSSION TITLE
Can't update private Image

### DISCUSSION THREAD

**Discussion by Touexe** [louislam/dockge#417] [Ask for Help] [answered]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I can't seem to update my container to latest image using the update button. I have confirmed that I've already logged in, and it works normally from `docker pull`

### 📝 Error Message(s) or Log

```
[+] Pulling 1/1
 ✘ website Error                                                                    0.5s 
Error response from daemon: Head "https://ghcr.io/v2/me/website/manifests/latest": unauthorized

```
### 🐻 Dockge Version

1.4.2

### 💻 Operating System and Arch

Ubuntu 22

### 🌐 Browser

Brave

### 🐋 Docker Version

25.0.2

**Comment by lohrbini** [comment, marked as answer]
Maybe adding the .docker path to the compose yaml could help since there are the credentials stored.
